source/lib/formatter.ts

```diff
@@ -33,6 +38,19 @@ export default (diagnostics: Diagnostic[]): string => {
 
 		entry.errorCount++;
 		entry.messages.push(diagnostic);
```

Comment:
Nitpicky, but I think adding the diagnostic should go after creating the diff, since L52 modifies `diagnostic.message`. (I know that `Map`s give you a mutable reference, but imo it's clearer that the mutation occurs before the `diagnostic` is added.)